pygeoapi/filters.py

```diff
@@ -0,0 +1,290 @@
+from operator import and_, or_, add, sub, mul, truediv
+# from datetime import datetime, timedelta
+# from functools import reduce
+
+
+def combine(sub_filters, combinator="AND"):
+    """ Combine filters using a logical combinator
+
+        :param sub_filters: the filters to combine
+        :param combinator: a string: "AND" / "OR"
+        :type sub_filters: list of filtered dict 
+        :return: the combined filter
+        :rtype: filtered dict
+    """
+    # check for sub_filters being the evaluation result of multiple sub_filters
```

Comment:
I think the assertions should be converted into a try/catch and raise exceptions eventually

Reply:
Done